packages/syft/src/syft/core/node/common/client_manager/dataset_api.py

```diff
@@ -174,6 +174,22 @@ def all_as_datasets(self) -> List[Any]:
             out.append(Dataset(raw, self.client, key=key, **a[key]))
         return out
 
+    def purge(self, skip_check: bool = False) -> None:
+        if not skip_check:
+            pref = input(
+                "You are about to delete all datasets ? 🚨 \n"
+                "All information will be permanantely deleted.\n"
+                "Please enter y/n to proceed: "
+            )
+            while pref != "y" and pref != "n":
+                pref = input("Invalid input '" + pref + "', please specify 'y' or 'n'.")
+            if pref == "n":
+                print("Datasets deletion is cancelled.")
+                return None
+
+        for dataset in self.all():
+            self.delete(dataset_id=dataset.get("id"))
```

Comment:
MAYBE: Shall we pass the skip check to .delete function from purge, so if a person passes a skip check, we could automatically skip check for the assets also ?